drivers/led/led_shell.c

```diff
@@ -84,6 +85,28 @@ static int cmd_on(const struct shell *sh, size_t argc, char **argv)
 	return err;
 }
 
+static const char *led_color(uint8_t color)
```

Comment:
I would also rename that into e.g `led_color_to_str`.

Reply:
Done